sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorExporterOptions.cs

```diff
@@ -1,12 +1,17 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System.ComponentModel;
+
 using Azure.Core;
 
 namespace Azure.Monitor.OpenTelemetry.Exporter
 {
     public class AzureMonitorExporterOptions : ClientOptions
     {
         public string ConnectionString { get; set; }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Don't add this property.

Reply:
Do we want to expose this property if it's not in use yet?

Reply:
Nope, removing it is fine.